conductor-client/src/main/kotlin/com/openlattice/assembler/Assembler.kt

```diff
@@ -250,8 +250,23 @@ class Assembler(
 
     fun createOrganization(organizationId: UUID, dbName: String) {
         createOrganizationTimer.time().use {
-            assemblies.set(organizationId, OrganizationAssembly(organizationId))
-            assemblies.executeOnKey(organizationId, InitializeOrganizationAssemblyProcessor(dbName).init(acm))
+
+            if (!assemblies.tryLock(organizationId)) {
```

Comment:
Pretty sure this will NPE b/c the key won't be in this map yet. May be better to move a portion of the logic in `acm.createAndInitializeOrganizationDatabase` into here or just create an entry in the organizations map and then do this try lock on that map

Reply:
ooh true